src/helpers/figures/charts/bar_chart.ts

```diff
@@ -96,8 +96,8 @@ export class BarChart extends AbstractChart {
     return {
```

Comment:
> The design options are still reset for now

This comment of the PR/commit is kinda wrong. We do preserve the background color for example. There's really no good reason not to also keep the other `design` options ? It's not more expansive to save them than the `config` options. I'd like to keep my axis to the right when switching from a line chart to a bar chart